.yamato/all-hdrp_performancetests.yml

```diff
@@ -0,0 +1,35 @@
+ 
+# This file is generated by .yamato/ruamel/build.py. Don't edit this file manually. 
+# Introduce any changes under .yamato/config/*.metafile files (for most cases) or under .yamato/ruamel/* within Python (more elaborate cases), and rerun build.py to regenerate all .yml files. 
+# Read more under .yamato/docs/readme.md 
+
+{% metadata_file .yamato/_latest_editor_versions_trunk.metafile -%}
+
+---
+
+PR_HDRP_PerformanceTests_trunk:
+    name: HDRP_PerformanceTests PR Job - trunk
+    dependencies: []
```

Comment:
Dependencies are empty for "all hdrp performance" config file, is this expected?

Reply:
I didn't add the tests part to any pipeline when starting this PR, I just focused on being able to run them manually. 
But good point, I added the memory test to the nightly in the last commit.
The Build editor one does not work, we'll have to investigate before adding it to the nightly.

Reply:
And I think we don't want to add it to the PR pipeline, only to the nightly. So it makes sense to keep the PR dependencies empty :)

Reply:
Ok, thanks :)